github_migration_process.md

```diff
@@ -0,0 +1,222 @@
+Proposed labels by section on GitHub
+====================================
+>
+Assignees
+---------------------------------------------------------------
+
+> No labels for this section - just use the alias of the individuals owning the issue. MS alias showing is preferred.
+
+Labels
+------------------------------------------------------------
+
+> It is understood that some teams will have their OWN GItHub page. Focus is on the SF page and we can build on/modify for other Team GitHub pages later once the initial model is completed.
+>
+>  
+### Label Rules for GitHub
+
+-   All tags and labels are lower-case and dash separated
+
+-   All hierarchy is whack separated and no caps.
+>  
+
+| ***State/Status*** | ***Area***               | ***Issue Type*** | ***Pri***       |
```

Comment:
Lets go with "status".  I think state feels more like it can only be in one.  I think it's possible there will be flows where you'll have multiple like `investigating`+`in-progress` or `investigating` and `needs-more-info`.  Ultimately I think we should just tell developers
> Use the status labels to define your issue as best you can

instead of having this huge document that attempts to predict and document every possible way the labels might be used.  I think if we trim this document to focus on what the labels mean and how they *might* be used, instead of how they *should* be used, we could have a lot more success.  That's my general advice for how this document could be reworded.